### ISSUE TITLE
[Discussion] LUA functions vs item names

### ISSUE THREAD

**Issue by racerand** [TeamPorcupine/ProjectPorcupine#765]
Me and @abackwood were just discussing how the naming system of `type_material` in snake_casing looks bad as a function name in LUA like `generator_oxygen_OnUpdate`. What do you guys think of this?

**Comment by bjubes** [comment]
I would keep Lua functions as they were with `OxygenGenerator_OnUpdate` As it makes more sense to have a capitalized function

**Comment by TomMalbran** [comment]
We can go for something like ```GeneratorOxygen_OnUpdate``` so that it keeps the furniture name order.

**Comment by racerand** [comment]
I like what @TomMalbran suggests

**Comment by TomMalbran** [comment]
@Grenkin1988 Should we change the names, or keep them as they are?

**Comment by Grenkin1988** [comment]
@TomMalbran They are not changed yet? This is 14 days old discussion without any updates. I guessed that this is done already.

**Comment by TomMalbran** [comment]
@Grenkin1988 No. We haven't even finished changing the furniture names in the xml yet.

**Comment by Grenkin1988** [comment]
OK. Than this needs to be done. And not just lost in time.

**Comment by TomMalbran** [comment]
I agree, but it will be easies once the Xml has the new standard names.

We should also add this standard somewhere in the none Lua standards.
